core/admin.py

```diff
@@ -125,9 +125,14 @@ class AllocationHistoryAdmin(admin.ModelAdmin):
     list_display = ('asset', 'current_owner', 'previous_owner', 'created_at')
 
 
+class AssetLogsAdmin(admin.ModelAdmin):
+    list_display = ('created_at', 'asset', 'checked_by', 'log_type')
```

Comment:
@HawiCaesar I'm thinking for the date, we should pick `last_modified` instead of `created_at`. Since that would give us the most recent date a log was saved. What are your thoughts on this @egichuri ?

Reply:
True @alexkiura, but in this case each asset log is its own record. Individual asset logs are not modified (`editable`) as per [here](https://github.com/AndelaOSP/art-backend/blob/develop/core/models/asset.py#L150). Both `created_at` and `last_modified` will contain the same data.

Reply:
True. True.